src/clr-addons/quick-list/add-option.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ClrQuickListValue } from './add-option.service';
+
+@Component({
+  selector: 'clr-add-option',
+  templateUrl: './add-option.html',
+  host: { '[class.quick-list-option]': 'true' },
+})
+export class ClrAddOption<T> {
```

Comment:
We would also need some unit tests for those two components please.

Reply:
I added unit tests only for QuickList component